StortSpelProjekt/Engine/src/Misc/AssetLoader.h

```diff
@@ -65,13 +74,11 @@ class AssetLoader
     DescriptorHeap* m_pDescriptorHeap_CBV_UAV_SRV = nullptr;
     Window* m_pWindow = nullptr;
     
-    // IsLoadedFunctions
-    bool IsModelLoadedOnGpu(const std::wstring& name) const;
-    bool IsModelLoadedOnGpu(const Model* model) const;
-    bool IsMaterialLoadedOnGpu(const std::wstring& name) const;
-    bool IsMaterialLoadedOnGpu(const Material* material) const;
-    bool IsTextureLoadedOnGpu(const std::wstring& name) const;
-    bool IsTextureLoadedOnGpu(const Texture* texture) const;
+    void loadDefaultMaterial();
+
+    // LoadVramFunctions
+    void loadMeshToGpu(Mesh* mesh);
+    void loadTextureToGpu(Texture* texture);
```

Comment:
These functions doesn't have a body, did you forget to remove?

Reply:
Ops, removed-